src/Visualization/Force.elm

```diff
@@ -344,7 +316,7 @@ manyBody =
 -}
 manyBodyStrength : Float -> List comparable -> Force comparable
 manyBodyStrength strength =
-    ManyBody 0.9 1 (1 / 0) << Dict.fromList << List.map (\key -> ( key, { strength = strength } ))
+    ManyBody 0 << Dict.fromList << List.map (\key -> ( key, { strength = strength } ))
```

Comment:
Is `0` really the best default value? AFAIK d3 uses `0.9` as their default for this.

Reply:
0 produces exactly the same result as the current implementation. Using a higher value is faster and still produces fine results, but users' output would suddenly change when they upgrade when a higher value is the default.

Reply:
OK, let's keep it for now, and I'll change it when 2.0 ships. Sounds OK?